CONTRIBUTING.md

```diff
@@ -350,11 +352,11 @@ To copy files over from Windows VM to your local machine, use Remote Desktop red
   connection to see the changes.
 1. Drag the executable/zip to the shared folder. You should now see it on your local machine.
 
-The zips should be called the following:
+The archives should be called the following:
```

Comment:
what you say archives are you suggesting only the RHEL 7 / 8 archives or all of the deliverables

I would change word to be "deliverables"

Reply:
changed